ZuluContent/Items/Misc/DragonEgg.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Server.Mobiles;
+
+namespace Server.Items
+{
+
+    public class DragonEgg : Item
+    {
+
+
+        public override double DefaultWeight
+        {
+            get { return 0.02; }
+        }
+
+
+        [Constructible]
+        public DragonEgg() : this(1)
+        {
+        }
+
+
+        [Constructible]
+        public DragonEgg(int amountFrom, int amountTo) : this(Utility.RandomMinMax(amountFrom, amountTo))
+        {
+        }
+
+
+        [Constructible]
+        public DragonEgg(int amount) : base(0x1725)
+        {
+            Stackable = true;
+            Amount = amount;
+            Name = "DragonEgg";
```

Comment:
This should be `Dragons Egg`, when looking at the POL `itemdesc.cfg` files the Name property is like the identifier, whereas the actual single click name is `desc`

https://github.com/zuluhotelaustralia/zha-2004-2006-side/blob/main/zhc93/pkg/opt/zuluitems/itemdesc.cfg#L125